vernac/declareDef.mli

```diff
@@ -59,6 +59,18 @@ val declare_assumption
   -> Entries.parameter_entry
   -> GlobRef.t
 
+val declare_mutual
```

Comment:
Not sure just `_mutual` is the best name. Maybe `_recursive` or `_mutual_recursive`?

Reply:
I kinda liked mutual as commonly say "a mutual definition" but I'm ok for `_mutual_recursive` [or should it be [_mutually_recursive]?

Reply:
Sure, let's pick one of `mutual_recursive` `mutual_definition` `mutual_recursive` `mutually_recursive` etc...